source/ecloud/private/cloning-and-templating-vms.md

```diff
@@ -1,6 +1,9 @@
 # Cloning and Templating VMs
 
 ```eval_rst
```

Comment:
Meta info needs to _not_ be at the start of the file otherwise search results don't render the page content snippet, e.g., https://docs.ukfast.co.uk/search.html?lang=None&q=cloning

Reply:
I think we need Tom to have a look at the elastic search populator in that case, right?

Reply:
Right this is why it was asked to move to the bottom

The content in the eval rst blocks is not returned by the search, but empty lines are, so if they can be stripped then search will work regardless of meta location

![image](https://user-images.githubusercontent.com/30502984/85727997-6b307c00-b6ef-11ea-979a-b1e65b0c5ea9.png)

I can see if thats code in here